test/__snapshots__/preset-attributify.test.ts.snap

```diff
@@ -41,37 +41,37 @@ exports[`attributify > autocomplete extractor 5`] = `
 exports[`attributify > compatible with full controlled rules 1`] = `
 "/* layer: default */
 
-  [custom~=\\"\\\\31 \\"] {
-    font-size: 1px;
+  .custom-2 {
+    font-size: 2px;
   }
   /* you can have multiple rules */
-  [custom~=\\"\\\\31 \\"]::after {
+  .custom-2::after {
```

Comment:
This seems not correct 🤔

Reply:
This is the breaking change noted above. The result was swapped with its second result to its bottom.

That particular rule is based on string return, so it will not have `selector` in `StringifiedUtil` hence sorted by its body. That portion of rule start with the captured token used as selector:
https://github.com/unocss/unocss/blob/0c861ebb8cb1fb39770955a7ac31e7f026438054/test/preset-attributify.test.ts#L87-L91
The string to extract was `<div custom="1" class="custom-2">` yielding `[custom~="1"]` and `.custom-2`. Sorting them leads to `.` > `[` hence the swapped result.